src/SDKs/Media/Management.Media/Properties/AssemblyInfo.cs

```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Media Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure Media.")]
 
-[assembly: AssemblyVersion("1.0.0")]
-[assembly: AssemblyFileVersion("1.0.0")]
+[assembly: AssemblyVersion("1.0.1")]
```

Comment:
AssemblyVersion must remain the same, AssemblyFileVersion must be updated

Reply:
Thanks, fixed.